tldw_Server_API/tests/Evaluations/conftest.py

```
@@ -148,7 +148,7 @@ def mock_openai_server():
     max_retries = 30
     for _ in range(max_retries):
         try:
-            response = requests.get("http://localhost:8080/health")
+            response = requests.get("http://localhost:8080/health")  # nosec B113
```

Ruff (0.15.1)

[error] Probable use of `requests` call without timeout

(S113)